exonum-light-client/src/main/java/com/exonum/client/ExonumClient.java

```diff
@@ -190,10 +190,12 @@ BlocksRange getLastBlocks(int size, BlockFilteringOption blockFilter,
    *
    * @param serviceName the name of a service instance
    */
-  Optional<ServiceInfo> getServiceInfoByName(String serviceName);
+  Optional<ServiceInfo> findServiceInfoByName(String serviceName);
```

Comment:
Maybe drop `ByName` since it must be clear enough from the parameter list and there are no other String query parameters (i.e., no `BySomeOtherStringProperty` that won't work with overloads)?